config/locales/en.yml

```diff
@@ -79,6 +79,7 @@ en:
     user_registrations:
       spree_user:
        signed_up_but_unconfirmed: "A message with a confirmation link has been sent to your email address. Please open the link to activate your account."
+       unknown_error: "Something went wrong while creating your account. Check your email address and try again."
```

Comment:
What originated this change was a system error, right? So, maybe we should mention it to the user. Like "Check your email address and try again. If the error persists please contact us/support."

Reply:
Good idea. What do you think about notifying Bugsnag instead of telling users to notify the support? We would then need to collect some errors that are associated with bad email addresses and ignore them.